content/references/rippled-api/public-rippled-methods/server-info-methods/server_state.md

```diff
@@ -274,6 +548,7 @@ The `state` object may have some arrangement of the following fields:
 | `state_accounting`               | Object           | A map of various [server states](rippled-server-states.html) with information about the time the server spends in each. This can be useful for tracking the long-term health of your server's connectivity to the network. [New in: rippled 0.30.1][] |
 | `state_accounting.*.duration_us` | String           | The number of microseconds the server has spent in this state. (This is updated whenever the server transitions into another state.) [New in: rippled 0.30.1][] |
 | `state_accounting.*.transitions` | Number           | The number of times the server has transitioned into this state. [New in: rippled 0.30.1][] |
+| `time`                           | String           | The current time in UTC, according to the server's clock. [Updated in: rippled 1.5.0][] |
 | `uptime`                         | Number           | Number of consecutive seconds that the server has been operational. [New in: rippled 0.30.1][] |
 | `validated_ledger`               | Object           | _(May be omitted)_ Information about the most recent fully-validated ledger. If the most recent validated ledger is not available, the response omits this field and includes `closed_ledger` instead. |
```

Comment:
If the most recent validated ledger is not available, the response omits this field and includes `closed_ledger` instead. - 
This suggests that it's possible for the response to both know there is a more recently validated ledger, and for it not to be available. Is it possible for the system to not know about both things? (I hope that question is clear. It's confusing to write out.)

Reply:
I'm not sure how to answer this question concisely.

But, basically, if the server is in sync with the network, then it provides this field with information about the most recent ledger that it knows to be validated.

Otherwise, the server provides the information about the most recent ledger version that it, locally, closed. That situation includes:

- The server knows that the network has validated a ledger that it doesn't have locally. (That's when you see ["We are not running on the consensus ledger" in the logs](https://xrpl.org/understanding-log-messages.html#we-are-not-running-on-the-consensus-ledger).)
- The server doesn't know if there's a consensus ledger or what the status of the network is, in general. (It probably just started syncing, or it may be totally cut off from the network.)

Does that answer your question?